docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -1,214 +1,215 @@
+Open the *Actions* menu and select *Edit integration* or *Delete integration*.
++
+Editing or deleting an integration is permanent and cannot be undone.
+Of course, if you make a mistake, you can always re-configure or re-add an integration.
+
+Any saved changes are immediately distributed and applied to all {agent}s enrolled in the given {policy}.
 
 [discrete]
-[[policy-main-settings]]
-== How to update a specific items in a {policy}
+[[copy-policy]]
+== Copy a policy
+
+Policy definitions are stored in plain-text yaml that can be downloaded or copied to another policy:
+
+. In {fleet}, click *Policies*.
+Click on the name of the policy you want to copy or download.
 
-Please note that separately, there are {policy} settings that are exposed on the 'settings' tab when
-viewing the given {policy}.  The settings there are high level toggle switches to enable or disable
-entire pieces of {agent} functionality and adjust overall {policy} values. The {policy} name can be
-modified here along with a given namespace. One of the high level toggles is the ability to set
-whether or not to collect Metrics on the {agent} process itself or  the desire for {agent}'s logs
-to be sent into {es} for monitoring.
+. To copy a policy, click *Actions* > *Copy policy*.
+Name the new policy, and provide a description.
+The exact policy definition will be copied to the new policy.
++
+Alternatively, view and download the policy definition by clicking *Actions* > *View policy*.
 
 [discrete]
-[[policy-actions]]
-== How to make use of {policy} level actions
-
-Finally, when viewing the {policy} view in {fleet}, a user can make a copy of or view the yaml text
-that makes up the {policy} definition.  To view the yaml details or copy a {policy}, click the
-'Actions' drop down on the top right of the UI and select your choice.  When viewing the policy yaml,
-you can see the current version of each Integration used. Each {policy} can have different versions
-of the Integration in use, but they will should be kept in sync (and up-to-date) except when first
-trying a new Integration version. The version information is listed under the 'inputs' section in
-the yaml, then under 'meta', 'package', and 'version'. Find the version that is directly below the
-relating package 'name' you are interested in.  This is useful when evaluating if you wish to update
-and evaluate using a newly released Integration version, discussed below.
+[[policy-main-settings]]
+== Edit or delete a policy
+
+High-level configurations like a policy's name, description, default namespace,
+and agent monitoring status can be changed as necessary:
+
+. In {fleet}, click *Policies*.
+Click on the name of the policy you want to edit or delete.
+
+. Click the *Settings* tab, make changes, and click *Save changes*
++
+Alternatively, click *Delete policy* to delete the policy.
+Existing data will not be deleted.
+Any agents assigned to a policy must be unenrolled before a policy can be deleted.
```

Comment:
we could make it more clear here with something like ...  "unenrolled, or simply assigned to a different policy, before a policy..."

Reply:
```suggestion
Any agents assigned to a policy must be unenrolled or assigned to a different policy before a policy can be deleted.
```